DnDDieRollCalculator/Program.cs

```diff
@@ -25,10 +25,11 @@ public async Task MainAsync()
             await Task.Delay(-1);
         }
 
-        private Task Log(LogMessage msg)
+        private Task LogUserMessage(LogMessage msg)
         {
             Console.WriteLine(msg.ToString());
             return Task.CompletedTask;
         }
+
```

Comment:
remove this extra line

Reply:
Let's also rename the folder name from DnDDieRollCalculator to TTCampaignManager
and Program.cs to DiscordBot.cs